src/app/GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -1313,13 +1313,13 @@ private void CheckoutToolStripMenuItemClick(object sender, EventArgs e)
 
         private void CommitToolStripMenuItemClick(object sender, EventArgs e)
         {
-            ForceActivate();
+            this.ForceActivate();
```

Comment:
Is not `this.` a warning, even removed in Designer files?

Reply:
Since this is an external method, this. is required.